python/pylibcudf/pylibcudf/tests/conftest.py

```diff
@@ -17,13 +17,23 @@
 from utils import ALL_PA_TYPES, DEFAULT_PA_TYPES, NUMERIC_PA_TYPES
 
 
-# This fixture defines the standard set of types that all tests should default to
+def _type_to_str(typ):
+    if isinstance(typ, pa.ListType):
+        return f"list-{_type_to_str(typ.value_type)}"
```

Comment:
I like `list[int]` more than `list-int` because it makes the hierarchy clear.
```suggestion
        return f"list[{_type_to_str(typ.value_type)}]"
```